Dockerfiles/agent/amd64/Dockerfile

```
@@ -115,6 +115,8 @@ ENV DOCKER_DD_AGENT=true \
     S6_BEHAVIOUR_IF_STAGE2_FAILS=2 \
     # Allow readonlyrootfs
     S6_READ_ONLY_ROOT=1
+    # Allow User Group to exec the secret backend script.
+    ENV DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM="true"
```

I’m not sure if `docker` would complain about this small indentation issue.

Doing this would be valid:
```suggestion
    S6_READ_ONLY_ROOT=1
    # Allow User Group to exec the secret backend script.
ENV DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM="true"
```

But this would create an extra layer. So maybe it would be better to do:
```suggestion
    S6_READ_ONLY_ROOT=1 \
    # Allow User Group to exec the secret backend script.
    DD_SECRET_BACKEND_COMMAND_ALLOW_GROUP_EXEC_PERM="true"
```

---

good catch, thank you @L3n41c !